CHANGELOG.md

```diff
@@ -23,6 +23,8 @@ x.x.x Release notes (yyyy-MM-dd)
   non-nil value when passed in an invalid URL.
 * `SyncSession.Progress.fractionTransferred` now returns 1 if there are no
   transferrable bytes.
+* Change the way sync progress notifications are dispatched to fix broken
```

Comment:
Yeah, good call. There was a bit more about this in the "Enhancements" section originally, but I deleted that part. Thanks for pointing out the omission.

> but we've always allowed ourselves to make breaking changes to already broken implementations

That was my rationale as well.